test/appium/tests/atomic/chats/test_one_to_one.py

```diff
@@ -636,3 +638,59 @@ def test_that_fiat_value_is_correct_for_token_transactions(self):
             self.errors.append('Wrong fiat value while requesting assets from wallet with user selected currency.')
 
         self.verify_no_errors()
+
+    @marks.testrail_id(5782)
+    @marks.high
+    def test_install_pack_and_send_sticker(self):
+        user = transaction_recipients['J']
+        sign_in = SignInView(self.driver)
+        home = sign_in.recover_access(user['passphrase'])
+        home.join_public_chat(home.get_public_chat_name())
+        chat = sign_in.get_chat_view()
+        chat.show_stickers_button.click()
+        chat.get_stickers.click()
+        chat.element_by_text('Install').click()
+        transaction_view = chat.get_send_transaction_view()
+        transaction_view.back_button.click()
+        time.sleep(2)
+        chat.swipe_left()
+        chat.sticker_icon.click()
+        chat.chat_item.is_element_displayed()
+
+    @marks.testrail_id(5783)
+    @marks.high
+    def test_purchase_pack_and_send_sticker(self):
```

Comment:
Test says test_puchase - but I don't see any purchase actions there. Only 'Install' and then transaction_view.back_button.click()
Is that right?